Swolify/swolify-fe
# Add return to homescreen and loss

# <p align="center">Swolify PR</p>
<hr>

## What will this PR do?
<hr>

- Adds a query which sends a Loss to BE upon quitting the game and going home.


## Where should the reviewer start?
<hr>

- sidebar.js


## How should this be manually tested?
<hr>

- Press the Swolify Logo without finishing a game, refresh and check loss graph from same user icon


## Any background context you want to provide?
<hr>

- n/a


## What are the relevant issues?
<hr>

- 


## Screenshots (if appropriate)
<hr>

-



## Pull request checklist:
<hr>

| Publisher: | Reviewer: |
| ---------- | --------- |
|<ul><li>- [x] I tested this locally for bugs and errors before pushing.</li><li>- [x] This code breaks no existing tests.</li><li>- [x] I have conducted a refactor on the code to remove comments and consoles.</li><li>- [x] I have requested someone to review my PR.</li></ul> | <ul><li>[x] I have run test files to confirm this does not fail any existing tests.</li><li>[x] I have reviewed all files edited</li><li>- [x] I have asked for clarifying information when I am unclear</li><li>- [x] This follows best practices and team decided norms.</li></ul>| 


## Questions:
<hr>

-
